src/main/java/jp/co/soramitsu/sora/json/Saltifier.java

```diff
@@ -0,0 +1,66 @@
+package jp.co.soramitsu.sora.json;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+@AllArgsConstructor
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+public class Saltifier {
+
+  ObjectMapper mapper;
+  SaltGenerator generator;
+  String valueKey;
+  String saltKey;
+
+  public Saltifier(ObjectMapper mapper, SaltGenerator generator) {
+    this(mapper, generator, "v", "s");
+  }
+
+  /**
+   * Add salt to the every *value* field in the input JSON.
```

Comment:
value field is every field except array?

but from example I see that they're saltified too:
```
+  "empty array" : {
+    "v" : [ ],
+    "s" : "salt"
+  },
```